test/common/run-tests

```diff
@@ -298,10 +298,10 @@ def get_affected_tests(test_dir, base_branch, test_files):
 
 
 def detect_tests(test_files, image, opts):
-    """Build the list of tests we'll parallelize and the ones we'll run serially"""
+    """Build the list of tests we'll run in parallel (destructive tests) and the ones we can run on the same machine (nondestructive)"""
```

Comment:
Perhaps "in separate machines" instead of "in parallel", as "parallel" is meaningless now. Also, this is too wide for a summary, please move that into the long description.